populus/config/glob.py

```diff
@@ -0,0 +1,9 @@
+from .base import Config
+
+
+class GlobalConfig(Config):
+
+    def __init__(self, config, parent=None, schema=None):
+
+        super(GlobalConfig,self).__init__(config,parent,schema)
```

Comment:
Also, I don't like the name `glob` for this module.  I'd prefer `global_.py` or to rename this from being the "Global" config to it being the "User" config since it's really not global, but the config that the user is using.